docs/enrich.rst

```diff
@@ -0,0 +1,113 @@
+.. _`enrich`:
+
+==================
+Enriching the data
+==================
+
+It might happen than the data returned by `AutoExtract API`_ is not
+enough: some additional information is required. It is possible
+to use regular selectors to enrich or amend the data provided by the
+API.
+
+Here we present an example of an spider that adds the UPC code to
+the extracted product.
+
+.. code-block:: python
+
+    import scrapy
+    from autoextract_poet.pages import AutoExtractProductPage
+
+    class ProductSpider(scrapy.Spider):
+
+        name = "products"
+        start_urls = ['https://books.toscrape.com/catalogue/sharp-objects_997/index.html']
+
+        def parse(self, response, product_page: AutoExtractProductPage):
+            product = product.to_item()
+            product._additional_attrs["UPC"] = response.css("tr:nth-child(1) td::text").get()
+            yield product
+
+Two things to comment in regards to this example:
+
+* The HTLM from the Scrapy response was used to extract the UPC. This means that
+  two requests were made to extract this product: one to the page itself using
+  Scrapy to obtain the response (HTML), and one to the AutoExtract API to extract
+  the product data.
+
+* The ``_additional_attrs`` dictionary is used to include additional attributes
+  not already defined in the :py:class:`autoextract-poet.items.Product`. It is an
+  easy way to add new attributes. We'll see later that using a custom pipeline
```

Comment:
I'm not sure this should be a recommended way though.

Reply:
I have changed the documentation. Now I show an example extending the Item, and only mention the unknown fields at the end. And saying that we don't recommend it.